framework/src/base/AuxiliarySystem.C

```diff
@@ -396,14 +405,17 @@ AuxiliarySystem::computeNodalVars(ExecFlagType type)
       solution().close();
       _sys.update();
     }
+    PARALLEL_CATCH;
+    Moose::perf_log.pop(compute_aux_tag, "Execution");
   }
-  PARALLEL_CATCH;
-  Moose::perf_log.pop("update_aux_vars_nodal()", "Execution");
 
-  // Boundary Nodal AuxKernels
-  Moose::perf_log.push("update_aux_vars_nodal_bcs()", "Execution");
-  PARALLEL_TRY {
-    if (nodal.hasActiveBoundaryObjects())
+  if (nodal.hasActiveBoundaryObjects())
```

Comment:
I can't image that they are any different across processors, the list of boundary/block stuff in the warehouses is populated from the 'boundary/block' input parameters. If those are not present then all blocks/boundaries are used from the mesh object.

I guess in that the complete list of blocks could be incomplete on a distributed mesh.